source/modulo_core/src/Cell.cpp

```diff
@@ -419,6 +425,28 @@ bool Cell::on_cleanup() {
 
 rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn Cell::on_shutdown(const rclcpp_lifecycle::State& state) {
   RCUTILS_LOG_INFO_NAMED(get_name(), "on_shutdown() is called from state %s.", state.label().c_str());
```

Comment:
maybe a `using namespace rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface` would be nice to reduce clutter for the `CallbackReturn`.